leasing/serializers/lease.py

```diff
@@ -151,6 +164,64 @@ class Meta:
         fields = ("type", "municipality", "district", "sequence")
 
 
+class TargetStatusSuccinctSerializer(serializers.ModelSerializer):
+    received_at = serializers.DateTimeField(source="answer.created_at")
+
+    class Meta:
+        model = TargetStatus
+        fields = (
+            "id",
+            "application_identifier",
+            "received_at",
+        )
+
+
+class PlotSearchSuccinctSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField()
+    name = serializers.CharField()
+    begin_at = serializers.DateTimeField()
+    end_at = serializers.DateTimeField()
+    type = serializers.CharField(source="subtype.plot_search_type.name")
+    subtype = serializers.CharField(source="subtype.name")
+
+    class Meta:
+        model = PlotSearch
+        fields = (
+            "id",
+            "name",
+            "begin_at",
+            "end_at",
+            "type",
+            "subtype",
+        )
+
+
+class AreaSearchSuccinctSerializer(serializers.ModelSerializer):
+    intended_use = serializers.CharField(source="intended_use.name")
+    applicant_names = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_applicant_names(obj):
+        answer = getattr(obj, "answer", None)
+        applicant_list = list()
+        if not answer:
+            return applicant_list
+        get_applicant(answer, applicant_list)
```

Comment:
I don't like these kind of functions that populate/change parameters, but since this is existing code and the function is used in multiple places already, I will just have to live with this. Better solution would be have a function (get_applicant) that would return applicant list which can be used however needed (in this case just returned directly to caller).

Reply:
I do agree with this but decided not to change it as it would have affected other parts of code which I would have needed to fix